proto/pocket/shared/service.proto

```diff
@@ -72,6 +72,7 @@ enum RPCType {
   WEBSOCKET = 2; // WebSocket
   JSON_RPC = 3; // JSON-RPC
   REST = 4; // REST
+  HYBRID = 5; // Hybrid (REST + JSON-RPC)
```

Comment:
I feel like this isn't "forward looking enough."

- If it's explicitly `REST + JSON-RPC`, let's name it `JSON_RPC_AND_REST`
- if it's "ambiguous", then let's call it `HYBRID` or `AMBIGUOUS`